### PR TITLE
[Candidate_list] integration test - complete skip test

### PR DESCRIPTION
**Pull request by kongtiaowang**
## Brief summary of changes
[Candidate_list] integration test - complete all skip tests, test react select dropdown menu by value not by order.
#### Testing instructions (if applicable)
Travis

### CODE REVIEW

File: modules/candidate_list/test/candidate_listTest.php [hunk 1]
```diff
@@ -244,22 +315,21 @@ function testDataEntryAndOpenProfile()
...
         $this->safeClick(WebDriverBy::cssSelector($btn));
-
         // input PSCID and DCCID
         $dccid = "#lorisworkspace > div > div:nth-child(1) > div >".
                  " div:nth-child(2)>form>div>div:nth-child(1)>div>div>input";
         $pscid = "#lorisworkspace > div > div:nth-child(1) > div >".
                  " div:nth-child(2)>form>div>div:nth-child(2)>div>div>input";
         // to do react input value
-        $this->webDriver->executeScript(
+        $this->webDriver->executescript(
```
Comment (lingma) [inline modules/candidate_list/test/candidate_listTest.php:324]: I just don't understand why we are modifying `executeScript` to `executescript` in most occurrences. I don't see any advantage but I am OK to accept, I suggest revert this kind of modifications.

File: modules/candidate_list/test/candidate_listTest.php [hunk 2]
```diff
@@ -244,22 +315,21 @@ function testDataEntryAndOpenProfile()
...
             "input = document.querySelector('$dccid');
                  lastValue = input.value;
                  input.value = '300001';
                  event = new Event('input', { bubbles: true });
                  input._valueTracker.setValue(lastValue);
                  input.dispatchEvent(event);"
         );
-        $this->webDriver->executeScript(
+        $this->webDriver->executescript(
```
Comment (lingma) [inline modules/candidate_list/test/candidate_listTest.php:332]: The same as above

File: modules/candidate_list/test/candidate_listTest.php (hunk 1 above)
Comment (kongtiaowang) [reply modules/candidate_list/test/candidate_listTest.php:324]: @lingma "executeScript" and "executescript" are the same in test cases. All of the other test cases use the "executescript", only these two are uppercase. That is why I changed it.

Comment (lingma) [reply modules/candidate_list/test/candidate_listTest.php:324]: I checked, currently not all is in uppercase, I don't see why we prefer a less advantaged solution.

**Pull request merged**